doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -73,50 +73,49 @@ sdk\<service name>\<package name>\<package name>.sln
 
   Make sure `npx cadl compile .` runs correctly.
 
-  ***Add cadl-csharp emitter***
-
-  Modify package.json, add one line under dependencies:
-
-```diff
-        "dependencies": {
-          "@cadl-lang/compiler": "^0.37.0",
-          "@cadl-lang/rest": "^0.19.0",
-          "@azure-tools/cadl-azure-core": "^0.9.0",
-+         "@azure-tools/cadl-csharp": "0.1.8"
-        },
-```
-
-        Run `npm install` again to install @azure-tools/cadl-csharp.
-
-  **Notes**: @azure-tools/cadl-csharp: "0.1.8" only works with @cadl-lang/compiler: "0.37.0" and @cadl-lang/rest: "0.19.0"
-  
-
-  ***Modify (or create) cadl-project.yaml, add one line under emitters:***
-
-  ```diff
-  emitters:
-+  "@azure-tools/cadl-csharp": true
-  ```
+  **Note**: It is recommended to configure Cadl Specs on [REST API specifications](https://github.com/Azure/azure-rest-api-specs). Please refer to the [Guidelines](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/cadl-structure-guidelines.md).
 
   ***Configuration (optional)***
```

Comment:
I think we should remove the word optional here as we will definitely to setup these configurations if we want to run the c# emitter.

Reply:
Some of these options are required and so we should definitely do them.

Reply:
updated